pkg/apis/iot/v1alpha1/platformadmin_conversion.go

```diff
@@ -36,7 +36,7 @@ func (src *PlatformAdmin) ConvertTo(dstRaw conversion.Hub) error {
 	dst.Spec.Version = src.Spec.Version
 	dst.Spec.Security = false
 	dst.Spec.ImageRegistry = src.Spec.ImageRegistry
-	dst.Spec.PoolName = src.Spec.PoolName
+	dst.Spec.NodePools = src.Spec.NodePools
```

Comment:
Instead of breaking the previous v1alpha1 and v1alpha2, we need a v1beta1 version here.